### ISSUE TITLE
LocalErrorResultStatus: path is not under exec_root

### ISSUE THREAD

**Issue by kschzt** [bazelbuild/reclient#23]
Trying to cross-compile Chromium on Linux for Windows, with buildbarn as RBE.

My clang-cl config is:
```
platform=OSFamily=Linux
labels=type=compile,compiler=clang-cl,lang=cpp
exec_strategy=remote
canonicalize_working_dir=true
remote_wrapper=/home/goma/bin/wrapper-win.sh
service_no_auth=true
service_no_security=true
```

The build fails (and wrapper is never called) with: 

`reclient[0e87d98a-887e-4286-bbda-43679fbb76de]: LocalErrorResultStatus: path C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include is not under C:\src\chromium\src\`

When building with goma, goma moves the SDK among other dependencies to the CAS before running the action. What am I missing? TIA

**Comment by gkousik** [comment]
@MikeS-rec has been looking into cross compilation for Chrome. Mike - thoughts?

**Comment by MikeS-rec** [comment]
Can you try adding this to your gn.args?

rbe_exec_root="C:\"

It's not an ideal solution (it might make getting remote cache hits less likely due to differences in paths to the chromium checkout across machines), but it would set the execution root to the root of the drive, so the build should be able to pick up the stuff in the visual studio directories.

**Comment by gkousik** [comment]
Reassigning to you @kschzt to confirm if Mike's suggestion above works.

**Comment by gkousik** [comment]
Closing this out due to lack of response. Feel free to reopen if you are still blocked by this.

**Comment by ekamwalia** [comment]
Hello @gkousik @MikeS-rec !

I am currently facing a similar problem. Since this one is closed, I filed a new issue here - https://github.com/bazelbuild/reclient/issues/34. Would you be able to help out? Thank you!

**Comment by yehia996** [comment]
Thank you
